src/interfaces.ts

```diff
@@ -401,6 +401,11 @@ export interface SharedOptions {
    * An EventEmitter instance to which we'll emit events.
    */
   events?: WorkerEvents;
+
+  /**
+   * A time source for getting "now", can be helpful in tests to artificially expire jobs 
+   */
+  now?: () => Date
```

Comment:
I don't think there's any need for this interface; instead use `jest-time-helpers` or similar and then you can just use `new Date()` as normal. I really don't want users to see `now` as an option in our configuration - it's purely a testing concern.

Instead let's say something like `useNodeTime` so it'll use the Node time instead of the Postgres time.

```suggestion

  /**
   * By default we use PostgreSQL's time source; in general this should be pretty close
   * (if not identical) to the time source your Node server is using, but in the case
   * that it isn't or you want to change it (e.g. in tests with fake date/time) you can
   * tell Worker to use Node's time source rather than Postgres' time source. (Default: 
   * false.)
   */
   useNodeTime?: boolean;
```